istioctl/cmd/webhook.go

```diff
@@ -0,0 +1,564 @@
+func Webhook() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "webhook",
+		Short: "webhook command to manage webhook configurations",
+	}
+
+	cmd.AddCommand(newEnableCmd())
+	cmd.AddCommand(newDisableCmd())
+	cmd.AddCommand(newStatusCmd())
+
+	return cmd
+}
+
+func newEnableCmd() *cobra.Command {
+	opts := &enableCliOptions{}
+	cmd := &cobra.Command{
+		Use:   "enable",
+		Short: "Enable webhook configurations",
+		Example: `
+# Enable the webhook configuration of Galley with the given webhook configuration
+istioctl experimental post-install webhook enable --validation --validation-secret istio.webhook.galley 
+    --namespace istio-system --config-path validatingwebhookconfiguration.yaml
+
+# Enable the webhook configuration of Galley with the given webhook configuration and CA certificate
+istioctl experimental post-install webhook enable --validation --validation-secret istio.webhook.galley 
+    --namespace istio-system --config-path validatingwebhookconfiguration.yaml --ca-bundle-file ./k8s-ca-cert.pem
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// --namespace is used as the namespace of the validation secret
+			opts.validationSecretNameSpace = namespace
+			if len(opts.webhookConfigPath) == 0 {
+				return fmt.Errorf("must specify a valid --config-path")
+			}
+			if len(opts.validationSecretName) == 0 {
+				return fmt.Errorf("must specify a valid --validation-secret")
+			}
+			if !opts.enableValidationWebhook {
+				return fmt.Errorf("not enabling validation webhook")
+			}
+			client, err := createInterface(kubeconfig)
+			if err != nil {
+				return fmt.Errorf("err when creating k8s client interface: %v", err)
+			}
+			// Read k8s CA certificate that will be used as the CA bundle of the webhook configuration
+			caCert, err := readCACert(client, opts.caCertPath, opts.validationSecretName, opts.validationSecretNameSpace,
+				opts.maxTimeForReadingWebhookCert)
+			if err != nil {
+				return fmt.Errorf("err when reading CA cert: %v", err)
+			}
+			fmt.Printf("Webhook CA certificate:\n%v\n", string(caCert))
+			// Apply the webhook configuration when the Galley webhook server is running.
+			err = waitForServerRunning(client, namespace, opts.validatingWebhookServiceName,
+				opts.maxTimeForCheckingWebhookServer)
+			if err != nil {
+				return fmt.Errorf("err when checking webhook server: %v", err)
+			}
+			webhookConfig, err := buildValidatingWebhookConfig(
+				caCert,
+				opts.webhookConfigPath,
+				opts.validatingWebhookConfigName,
+			)
+			if err != nil {
+				return fmt.Errorf("err when build validatingwebhookconfiguration: %v", err)
+			}
+			_, err = createValidatingWebhookConfig(client, webhookConfig)
+			if err != nil {
+				return fmt.Errorf("error when creating validatingwebhookconfiguration: %v", err)
+			}
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVar(&opts.enableValidationWebhook, "validation", true, "Specifies whether enabling"+
```

Comment:
Instead of `"Specifies whether enabling the validating webhook.` use `Enable validating webhook (default true)`.

Reply:
Done.